quench_linac.py

```diff
@@ -127,4 +133,20 @@ def validate_quench(self, wait_for_update: bool = False):
         return is_real
 
 
-QUENCH_CRYOMODULES = scLinac.CryoDict(cavityClass=QuenchCavity)
+class QuenchCryomodule(scLinac.Cryomodule):
+    def __init__(self, cryo_name, linac_object, cavity_class=QuenchCavity,
+                 magnet_class=scLinac.Magnet, rack_class=scLinac.Rack,
+                 is_harmonic_linearizer=False, ssa_class=scLinac.SSA,
+                 stepper_class=scLinac.StepperTuner, piezo_class=scLinac.Piezo):
+        super().__init__(cryo_name, linac_object, cavity_class=QuenchCavity)
+        self.logger = logging.getLogger(f'{self} quench resetter')
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
Does each log entry have a timestamp? That would be helpful

Reply:
No! Good point